<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomFeatures_Edit</class>
 <widget class="QDialog" name="RoomFeatures_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Room Features</string>
  </property>
  <widget class="QListWidget" name="rmfList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureNameEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Feature name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="featureDeacriptionEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>230</width>
     <height>140</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Feature description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Some information here about something</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>displayItemInformation(QListWidgetItem*)</slot>
  <slot>displayItemChanged(QString)</slot>
 </slots>
</ui>
